src/libaktualizr/config/config.cc

```diff
@@ -258,10 +256,21 @@ void Config::readSecondaryConfigs(const boost::filesystem::path& sconfigs_dir) {
     LOG_ERROR << "Could not read secondary configs from " << sconfigs_dir << ": not a directory";
     return;
   }
+#if (!defined(ANDROID) || __ANDROID_API__ >= 28)
   for (const auto& config_file : Utils::glob((sconfigs_dir / "*.json").string())) {
```

Comment:
Can we use the more generic `directory_iterator`-based implementation for both cases and get rid of external compilation here?

Reply:
I was unsure about Utils::glob purpose. It could be definitely changed to iterator-based implementation if its only aimed to select entries by extension.